modules/sdk/project-templates/project-templates-rest-builder/src/main/java/com/liferay/project/templates/rest/builder/internal/RESTBuilderProjectTemplatesArgs.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.project.templates.rest.builder.internal;
+
+import com.beust.jcommander.Parameter;
+
+import com.liferay.project.templates.extensions.ProjectTemplatesArgsExt;
+
+/**
+ * @author Javier de Arcos
+ */
+public class RESTBuilderProjectTemplatesArgs
+	implements ProjectTemplatesArgsExt {
+
+	public String getExtraModules() {
+		return _extraModules;
+	}
+
+	@Override
+	public String getTemplateName() {
+		return "rest-builder";
+	}
+
+	public void setExtraModules(String extraModules) {
+		_extraModules = extraModules;
+	}
+
+	@Parameter(
+		description = "Specify if client and test integration modules should be created.",
+		names = "--extraModules"
```

Comment:
Do we have documentation on what the `client` and `test` modules are and how they are to be used?
I think `extraModules` is not a good name for these two modules, but I cannot provide a better name without a better understanding of what they are for.

Reply:
After further discussion, i think it might better if we just generate everything and let the user decide if they want it or not.

Reply:
Ok, I'll revert the changes including the extraModules parameter and create a new PR, but I see that you already forwarded to @petershin... How do you want me to proceed?

Reply:
nothing you need to do for this review at least. Thanks for the work!